README.md

```diff
@@ -1,88 +1,78 @@
-# Project Title
+# Task-Force---Employee-management-REST-API-Back-End-3
 
-One Paragraph of project description goes here
+This is a Task-Force challenge number 3 which was to create an employee management REST API.
 
 ## Getting Started
 
-These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See deployment for notes on how to deploy the project on a live system.
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See deployment for notes on how to deploy the project on your own live system.
 
 ### Prerequisites
 
-What things you need to install the software and how to install them
+You need to have:
 
-```
-Give examples
-```
+-   NodeJs Runtime if not [download it here](https://nodejs.org/en/)
+-   PostgreSQL Database if not [download it here](https://www.enterprisedb.com/downloads/postgres-postgresql-downloads)
+-   Git Version Control if not [download it here](https://git-scm.com/downloads)
 
-### Installing
+To check The Prerequisites are installed you can use these terminal commands:
 
-A step by step series of examples that tell you how to get a development env running
+For NodeJs:
 
-Say what the step will be
+> node --version
 
-```
-Give the example
-```
-
-And repeat
+For Postgres:
 
-```
-until finished
-```
+> psql --version
 
-End with an example of getting some data out of the system or using it for a little demo
+For Git:
 
-## Running the tests
+> git --version
 
-Explain how to run the automated tests for this system
+### Installing
 
-### Break down into end to end tests
+A step by step series of examples that tell you how to get a development env running
 
-Explain what these tests test and why
+Clone the Repo with the terminal command:
 
 ```
-Give an example
+git clone https://github.com/RedJanvier/Task-Force---Employee-management-REST-API-Back-End-3.git
 ```
 
-### And coding style tests
+then make a file called .env using sample.env by replacing with your own data.
```

Comment:
some exact commands and details would be more helpful but as is; I think it is the most it can be.